src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__gradebook_audit_flags.sql

```diff
@@ -1,42 +1,7 @@
-{{ config(materialized="table", cluster_by="cte_grouping") }}
-
+-- {{ config(materialized="table") }}
```

Comment:
I think you need to delete this line entirely. dbt will still read it as a valid config.